modules/web-console/bin/web-console.bat

```diff
@@ -82,6 +82,18 @@ goto checkIgniteHome2
 
 :run_java
 
+::
+:: JVM options. See http://java.sun.com/javase/technologies/hotspot/vmoptions.jsp for more details.
+::
+:: ADD YOUR/CHANGE ADDITIONAL OPTIONS HERE
+::
+if "%JVM_OPTS%" == "" set JVM_OPTS=-Xms1g -Xmx1g -server -XX:MaxMetaspaceSize=256m
+
+::
+:: Disable update notifier.
+::
+set JVM_OPTS=-DIGNITE_UPDATE_NOTIFIER %JVM_OPTS%
+
```

Comment:
Should -DIGNITE_UPDATE_NOTIFIER be equal false as in sh file?

Reply:
Nice catch, fixed